projects/storefrontlib/src/cms-components/user/login-form/login-form.component.ts

```diff
@@ -59,7 +59,7 @@ export class LoginFormComponent implements OnInit, OnDestroy {
       password: ['', Validators.required],
     });
 
-    if (this.featureConfig.isEnabled('guestCheckout')) {
+    if (this.checkoutConfigService.isGuestCheckout()) {
```

Comment:
Before invoking that method check if this service is provided? It is optional, so there is a chance that it won't be provided.